documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -1,5 +1,12 @@
 # Current Release notes
 
+## Version 8.11.2
+
+### New features
+
+* Adds logic to pull necessary artifacts from the repo.blackduck.com artifactory. If this is not accessible then artifacts will be downloaded from the sig-repo.synopsys.com repository. 
```

Comment:
I believe we are to avoid the use of the term "artifactory" with the new repo. Use the term "repository" instead?  (Lines 7 & 8)

Reply:
I would expand this suggestion to code as well.

Reply:
Pushed a new commit. I changed the release notes as those are customer facing. I'm hesitant to change at the code level as there are lots of places that use the artifactory term and I'd rather not do a large refactor since that is mostly internal.